File: go/problem1.go
package problem1

import (
	"fmt"
)

func main() {

	fmt.Println(SumOfMultiplesOf3And5(10))
}

func SumOfMultiplesOf3And5(limit int) int {

	result := 0
	for i := 1; i < limit; i++ {
		if i%3 == 0 || i%5 == 0 {
			result += i
		}
	}

	return result
}

File: python/problem1.py
def sumOfMultiplesOf3And5(limit):
    sum = 0
    for x in list([n for n in range(1,limit) if n % 3 == 0 or n % 5 == 0]):  #numbers:
        sum += x
    return(sum)

if __name__ == '__main__':
    print(sumOfMultiplesOf3And5(10))
